### ISSUE TITLE
Fix `aria-allowed-attr` violation 

### ISSUE THREAD

**Issue by DanielCliftonGuardian** [guardian/dotcom-rendering#15067]
WCAG: 2.0 A (4.1.2)
Affected: 1 story
Fix: Remove unsupported ARIA attributes from elements

Affected Components
Discussion/Abuse Report Form - Dialog
